src/navigation/components/groupDraftItem/GroupDraftItem.test.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import GroupDraftItem from './GroupDraftItem';
+
+const MOCK_PROPS = {
+	groupHome: jest.fn(),
+	groupDraft: {
+		urlname: 'meetup.com/groups',
```

Comment:
I don't _think_ `urlname` will generally have the domain included - probably better to use a more accurate example.